### ISSUE TITLE
[🐛 Bug]: malformed python wheel

### ISSUE THREAD

**Issue by jjangga0214** [SeleniumHQ/selenium#10022]
### What happened?

Selenium v4's python wheel is malformed.
REF: https://github.com/pdm-project/pdm/issues/719#issuecomment-961758667

### How can we reproduce the issue?

```shell
.
```

### Relevant log output

```shell
.
```

### Operating System

Linux

### Selenium version

4

### What are the browser(s) and version(s) where you see this issue?

.

### What are the browser driver(s) and version(s) where you see this issue?

.

### Are you using Selenium Grid?

No

**Comment by titusfortner** [comment]
Is this still an issue for Selenium 4.1?

**Comment by titusfortner** [comment]
This is reproduction:

```
wheel unpack selenium-4.1.0-py3-none-any.whl 
```

which results in:
```
Unpacking to: ./selenium-4.1.0...No hash found for file 'selenium/__init__.py'
```

@AutomatedTester do we need to get bazel to run this with `--require-hashes`?

**Comment by mhsmith** [comment]
I think this is caused by the RECORD file in the wheel incorrectly containing leading slashes on every filename, like this:

```
/selenium/__init__.py,sha256=l8nEsTP4D2dZVula_p4ZuCe8AGnxOq7MxMeAWNvR0Qc,811
```

**Comment by sebthom** [comment]
Can this please get some attention. 🙏

**Comment by AutomatedTester** [comment]
Are we expecting a `RECORD` file to contain
```
selenium-4.1.2.dist-info/METADATA,sha256=ODvTAqLkXp3JPOjcURvS6gVQIstligNoioi_VdiAsSw,6475
selenium-4.1.2.dist-info/RECORD,,
selenium-4.1.2.dist-info/WHEEL,sha256=sobxWSyDDkdg_rinUth-jxhXHqoNqlmNMJY3aTZn2Us,91
selenium/__init__.py,sha256=u_ypJxZ63ySnXX8ikn3QfoxJakG0WmPwC1iD0L_gjfs,811
selenium/common/__init__.py,sha256=E_V8-VsmWsxJhOnXZn-72u9k_jCgBI6PjV7kEd7P8jA,821
selenium/common/exceptions.py,sha256=oZx2PS-g1gYLqJA_oqzE4Rq4ngplqlwwRBZDofiqni0,9309
selenium/py.typed,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
selenium/types.py,sha256=U6jBMYQzZUIsZ4MYrWCknaTMyL0DHJ3_ZMsX_ZfBe8I,881
selenium/webdriver/__init__.py,sha256=SpGOo5oOauglH5UkqgJmdX1H8ac-ZH0g6xqMRzp9BZ4,2025
selenium/webdriver/chrome/__init__.py,sha256=3TKaBBK08eiCsGGFFcZlZwwjHHcmj2YO0xImghpJk38,787
selenium/webdriver/chrome/options.py,sha256=2I-RKskaWPdRdh-fwK8Pc23EwMVlyvkfzJHJWZ054Ss,1430
selenium/webdriver/chrome/service.py,sha256=kyEOO-Uo47yWkXt3J_ykYvbG4LPkEAtVhUzzCq18EvU,1763
selenium/webdriver/chrome/webdriver.py,sha256=EO56zuMnah8WrhZ7q1uvQSsrZd-SJpcZWgy9HMB_NQQ,3716
selenium/webdriver/chromium/__init__.py,sha256=3TKaBBK08eiCsGGFFcZlZwwjHHcmj2YO0xImghpJk38,787
selenium/webdriver/chromium/options.py,sha256=EHFa4XkO_c4ck_bYcjmmcBc9o6q-BB_LX4EKfigRr74,6196
selenium/webdriver/chromium/remote_connection.py,sha256=NirPbNkYG-AA6Low_QgEiNHgd5DjAjGK0eDDfco6vzA,2515
selenium/webdriver/chromium/service.py,sha256=xz2bRd4nSBsJIx-Nb82XD4nyKFs999CXjhOLGY5JfFw,1991
selenium/webdriver/chromium/webdriver.py,sha256=ioYLbYQZeO3ELO6XXgTX-EyzPI-aYJvEnJUQu7DyVtA,9475
selenium/webdriver/common/__init__.py,sha256=3TKaBBK08eiCsGGFFcZlZwwjHHcmj2YO0xImghpJk38,787
selenium/webdriver/common/action_chains.py,sha256=S9pomlfvywk3UbDHFZZ6U0vWyj1vDu87yf_n_5WeCg4,10735
selenium/webdriver/common/actions/__init__.py,sha256=3TKaBBK08eiCsGGFFcZlZwwjHHcmj2YO0xImghpJk38,787
```

where at the moment it's containing a leading slash on the file names?

Not having spent a lot of time reading the PEP for building wheels a little guidance here would be good. The PDM issue, or the initial issue doesn't give me hints about what it should be.

**Comment by AutomatedTester** [comment]
```bash
davidburns in ~/development/test_pdm λ pdm add ../selenium/bazel-bin/py/selenium-4.1.2-py3-none-any.whl
Adding packages to default dependencies: selenium @ file:///private/var/tmp/_bazel_davidburns/ee760a273fbaa30310e4e04981835b81/execroot/selenium/bazel-out/darwin-fastbuild/bin/py/selenium-4.1.2-py3-none-any.whl
✔ 🔒 Lock successful
Changes are written to pdm.lock.
Changes are written to pyproject.toml.
Synchronizing working set with lock file: 18 to add, 0 to update, 0 to remove

  ✔ Install async-generator 1.10 successful
  ✔ Install attrs 21.4.0 successful
  ✔ Install certifi 2021.10.8 successful
  ✔ Install cffi 1.15.0 successful
  ⠙ Installing cryptography 36.0.1...
  ⠹ Installing cryptography 36.0.1...
  ⠸ Installing cryptography 36.0.1...
  ⠼ Installing cryptography 36.0.1...
  ⠴ Installing cryptography 36.0.1...
  ✔ Install cryptography 36.0.1 successful
  ✔ Install outcome 1.1.0 successful
  ✔ Install h11 0.13.0 successful
  ✔ Install idna 3.3 successful
  ✔ Install pycparser 2.21 successful
  ✔ Install pyopenssl 22.0.0 successful
  ✔ Install pysocks 1.7.1 successful
  ✔ Install selenium 4.1.2 successful
  ✔ Install sniffio 1.2.0 successful
  ✔ Install sortedcontainers 2.4.0 successful
  ✔ Install trio 0.20.0 successful
  ✔ Install trio-websocket 0.9.2 successful
  ✔ Install urllib3 1.26.8 successful
  ✔ Install wsproto 1.1.0 successful

🎉 All complete!
```

Looks like it was a silly mistake in Bazel.

**Comment by AutomatedTester** [comment]
I will get a release out soon

**Comment by ElijahSink** [comment]
Any update here? This is currently holding back some CI pipelines for me.

**Comment by titusfortner** [comment]
@ElijahSink just released 4.1.3
